Cliques.js

```diff
@@ -62,9 +59,12 @@ Clique.prototype.onMessage = function(sender, message) {
 		payload = JSON.parse(message)
 		payload.id = sender.id
 		payload.name = sender.name
+
 		switch (payload.event) {
 			case 'friend-updated':
-				sender.position = payload.position
+				sender.position = !payload.position ? sender.position : payload.position
```

Comment:
This line is unfortunately necessary as of now because the only time the position is broadcasted is on updates. When the user goes stale, we do not have access to their previous position(at least I don't think so).